app/views/mail/extendPricingPlan.scala.html

```diff
@@ -0,0 +1,8 @@
+@( name: String, organizationDisplayName: String)
```

Comment:
Iwie ist das mit den emails weird. Wollen wir vllt 2 emails senden:
- kurze Email mit Bullet points an uns:
   - Upgrade request: Subscription renewal
   - Current subscription end: ...
   - User: ...
   - Organization: ...
 - eine nette bestätigungs email an den user:
   - Thank you for contacting us. You have requested to extend your webKnossos account until .... We will be in touch shortly for the next steps.

Reply:
Bei 2 Emails können wir halt nicht einfach auf Reply drücken.

Reply:
Ne, die eine Email hat ja dann den Empfänger nicht mehr in cc oder?

Reply:
hab jetzt eine "Confirmation" email an den User gemacht und eine Email an Sales